.github/ISSUE_TEMPLATE/bug_report.yml

```diff
@@ -0,0 +1,94 @@
+name: "\U0001F41B Bug Report"
+description: "Report a bug or problem with the code."
+title: "(short issue description)"
+labels: [bug, needs-triage]
+body:
+  - type: textarea
+    id: description
+    attributes:
```

Comment:
use placeholder property to guide customer what could be a potential value for each type.